### PR TITLE
Streamline misc

### PR DESCRIPTION
**Pull request by BBC-Esq**
Some minor attempts to streamline

### CODE REVIEW

File: koboldcpp.py [hunk 1]
```diff
@@ -2582,21 +2578,24 @@ def pickpremadetemplate():
...
+        
+        for item, label in horde_items:
             if usehorde_var.get() == 1:
                 item.grid()
-                labels[idx].grid()
+                label.grid()
             else:
                 item.grid_remove()
-                labels[idx].grid_remove()
-        if usehorde_var.get()==1 and (horde_name_var.get()=="koboldcpp" or horde_name_var.get()=="") and model_var.get()!="":
```
Comment (LostRuins) [inline koboldcpp.py:2593]: Why did you remove koboldcpp

Comment (LostRuins) [reply koboldcpp.py:2593]: I'll merge and fix this separately

Comment (BBC-Esq) [reply koboldcpp.py:2593]: Thanks, accidental of course.
